rust/cardano-chain-follower/examples/follow_chains.rs

```diff
@@ -242,24 +232,6 @@ async fn follow_for(network: Network, matches: ArgMatches) {
             break;
         }
 
-        // Inspect the transactions in the block.
-        for (tx_idx, _tx) in block.txs().iter().enumerate() {
-            if let Some(aux_data) =
-                update_biggest_aux_data(&chain_update, tx_idx, largest_metadata, biggest_aux_data)
-            {
-                biggest_aux_data = aux_data;
```

Comment:
Can we still track the biggest `aux_data` block.  This is uesful from a blockchain analysis perspective.  Which is why I had it.